sql/testdata/datetime

```diff
@@ -17,7 +17,7 @@ INSERT INTO t VALUES
 query TTT
 SELECT * FROM t WHERE a = '2015-08-25 05:45:45.53453+01:00'::timestamp
 ----
-2015-08-25 04:45:45.53453+00:00 2015-08-25 2h45m2.234s
+2015-08-25 04:45:45.53453 +0000 UTC   2015-08-25 00:00:00 +0000 UTC   2h45m2.234s
```

Comment:
Do we accept this format (with spaces and "UTC") on input? We can do whatever we want for tests, but for the user-facing CLI our output format should match our canonical input format. Add a TODO to replace `fmt` with a custom output function (in the CLI).